### ISSUE TITLE
Dedicated Task Queues

### ISSUE THREAD

**Issue by jokiefer** [mrmap-community/mrmap#25]
With an increasing number of apps using the background workers, we should think about creating dedicated queues for certain tasks. This should ensure that crucial tasks will not be blocked by time consuming lower priority tasks (e.g. metadata validation).

I suggest we should at least create dedicated queues for metadata validation and service registration, respectively.

What do you think @hollsandre ?

**Comment by jokiefer** [comment]
Yes this is plausible to me!
I assume this is achieved by code changes only and the celery configuration can stay as it is?

**Comment by jokiefer** [comment]
I did not check yet on how to enable this feature.

It might require some changes on the celery configuration (e.g. assigning worker pools).

**Comment by jokiefer** [comment]
implemented
